docs/api/converters.rst

```diff
@@ -0,0 +1,75 @@
+Downloaders
+===========
+
+General converters
+------------------
+
+.. automodule:: fuel.converters.base
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+Adult dataset converter
```

Comment:
I'd remove `dataset converter` from subsection titles in this file, once again because of redundancy.